src/client/dfuse/ops/read.c

```diff
@@ -58,44 +57,39 @@ dfuse_cb_read(fuse_req_t req, fuse_ino_t ino, size_t len, off_t position, struct
 		mock_read = true;
 	}
 
-	D_ALLOC(buff, len);
-	if (!buff)
-		D_GOTO(err, rc = ENOMEM);
+	if (len > ev->de_iov.iov_buf_len) {
+		D_ERROR("Fuse read buffer not large enough %zx > %zx\n", len,
+			ev->de_iov.iov_buf_len);
+	}
```

Comment:
In practice, when will this happen?

Reply:
When the kernel changes to support larger buffers.